### PR TITLE
[EX-6] Implement peers in Jellyfish ios client sdk

### PR DESCRIPTION
**Pull request by skyman503**

### CODE REVIEW

File: JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift [hunk 1]
```diff
@@ -12,6 +12,40 @@ public struct Config {
...
+internal protocol JellyfishWebsocket {
+    var delegate: WebSocketDelegate? { get set }
+    func connect()
+    func disconnect()
+    func write(data: Data)
+}
+
+public class JellyfishClientWebSocket: JellyfishWebsocket {
+  var socket: WebSocket
+  var delegate: WebSocketDelegate? { set{self.socket.delegate = newValue} get{self.socket.delegate}}
```
Comment (graszka22) [inline JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift:24]: why a getter/setter here if it doesn't do anything than getting and setting a field? can't we just use that field directly?

File: JellyfishClientSdk/project.json [hunk 2]
```diff
@@ -52,7 +55,7 @@
   ],
   "manifest_display_name" : "JellyfishClientSdk",
   "name" : "JellyfishClientSdk",
-  "path" : "/Users/angelikaserwa/Projects/ios-client-sdk/JellyfishClientSdk",
+  "path" : "/Users/karolsygiet/sygiet/work/membrane/ios-client-sdk/JellyfishClientSdk",
```
Comment (graszka22) [inline JellyfishClientSdk/project.json:58]: this is weird, if there is a local path maybe this shouldn't be checked in the repo or there is something else wrong?

**Review by graszka22 — APPROVED** [review]
LGTM, good job on figuring out the sockets and tests! 
I have only two suggestions

File: JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift (hunk 1 above)
Comment (skyman503) [reply JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift:24]: In `JellyfishClientInternal` line 21 the delegate is assigned to our socket but in fact we want to assign it to the real socket  underneath, it was a faster way to write this, I can change it if you want

File: JellyfishClientSdk/project.json (hunk 2 above)
Comment (skyman503) [reply JellyfishClientSdk/project.json:58]: huh the webrtc repo does not have this file +  it is also not present in its gitignore idk what to do about that

File: JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift (hunk 1 above)
Comment (graszka22) [reply JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift:24]: aah ok, let's keep it

File: JellyfishClientSdk/project.json (hunk 2 above)
Comment (graszka22) [reply JellyfishClientSdk/project.json:58]: this looks like mockingbird stuff https://mockingbirdswift.com/spm-package-quickstart they don't recommend gitignoring it so let's keep it for now 🤷‍♀️

**Review by graszka22 — APPROVED** [review]

**Pull request merged**